src/UnitySampleProject/Assets/Samples/web3.unity SDK/2.5.0/Web3.Unity Samples/Scripts/Scenes/SampleMain/EVM/EvmCalls.cs

```diff
@@ -0,0 +1,277 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Transactions;
+using ChainSafe.Gaming.MultiCall;
+using ChainSafe.Gaming.UnityPackage;
+using ChainSafe.Gaming.Web3;
+using Nethereum.Contracts.QueryHandlers.MultiCall;
+using Nethereum.Hex.HexConvertors.Extensions;
+using Scripts.EVM.Token;
+using UnityEngine;
+using Web3Unity.Scripts.Prefabs;
+using ABI = ChainSafe.Gaming.UnityPackage.ABI;
+
+public class EvmCalls : MonoBehaviour
+{
+    #region Fields
+    
+    #region IPFS
+
+    private string apiKey = "YOUR_CHAINSAFE_STORE_API_KEY";
+    private string data = "YOUR_DATA";
+    private string bucketId = "BUCKET_ID";
+    private string path = "/PATH";
+    private string filename = "FILENAME.EXT";
+
+    #endregion
+    
+    # region Contract Send
+        
+    private string methodSend = "addTotal";
+    private string abiSend = "[ { \"inputs\": [ { \"internalType\": \"uint8\", \"name\": \"_myArg\", \"type\": \"uint8\" } ], \"name\": \"addTotal\", \"outputs\": [], \"stateMutability\": \"nonpayable\", \"type\": \"function\" }, { \"inputs\": [], \"name\": \"myTotal\", \"outputs\": [ { \"internalType\": \"uint256\", \"name\": \"\", \"type\": \"uint256\" } ], \"stateMutability\": \"view\", \"type\": \"function\" } ]";
```

Comment:
This ABI appears in other files

Reply:
yeah, it's just there to show users what variables to add directly

Reply:
I see your point @sneakzttv, I guess it's okay for samples but we should have a single point of ref for ABIs imo

Reply:
updated